src/pivot-table/components/cells/DimensionTitleCell.tsx

```diff
@@ -11,7 +11,7 @@ import { useStyleContext } from "../../contexts/StyleProvider";
 import type { GetHeaderCellsIconsVisibilityStatus, OverrideLeftGridWidth } from "../../hooks/use-column-width";
 import { useHeadCellDim } from "../../hooks/use-head-cell-dim";
 import { baseCellStyle, getBorderStyle } from "../shared-styles";
-import ColumnAdjuster from "./ColumnAdjuster";
+import ColumnAdjuster from "./ColumnAdjusterWrapper";
```

Comment:
I think it's better to keep using `ColumnAdjusterWrapper` instead, 
if we try to search ColumnAdjuster -> it refers to 2 things now: 
1. this one (which wraps the actual adjuster from shared lib)
2. the one from shared lib

Same applies for `DimentionsValue/index` and the test file as well!

Reply:
you mean the name of the import?